integration-tests/common/test_common.go

```diff
@@ -15,13 +15,12 @@ import (
 	"github.com/google/uuid"
 	"github.com/lib/pq"
 	"github.com/rs/zerolog/log"
-	"github.com/stretchr/testify/require"
-	"gopkg.in/guregu/null.v4"
-
 	test_env_ctf "github.com/smartcontractkit/chainlink-testing-framework/lib/docker/test_env"
 	"github.com/smartcontractkit/chainlink-testing-framework/lib/utils/testcontext"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/guregu/null.v2"
```

Comment:
Why downgrade this from v4?

Reply:
Maybe consider updating the linter file like we do in core:
https://github.com/smartcontractkit/chainlink/blob/50c1b3dbf31bd145b312739b08967600a5c67f30/.golangci.yml#L139-L144

Reply:
Weird, I don't know why that keeps happening. Each time I notice it changes to `null.v2` I put it back to `null.v4` but... something I'm running really wants it to be v2.  Must be either goland or vim that's changing it?

Reply:
Hmm... it looks like we do have those linter rules already in `chainlink-solana/.golangci.yml`. Maybe we need to create a separate `chainlink-solana/integration-tests/.golangci.yml` or do the existing linter rules apply to both modules?

Reply:
Looks like the existing yml file does apply to both, because the `make lint-go-integration-tests` command is run from `chainlink-solana` root dir

Reply:
Oh, I see why this null.v2 problem came back... I resolved some merge conflicts while rebasing `go.mod` by running `git rebase --skip`, apparently I undid the fix for that and some related things.